src/utils/sounds.js

```diff
@@ -22,7 +22,7 @@ import { generateFilePath } from '@nextcloud/router'
 import store from '../store'
 
 export const Sounds = {
-	BLOCK_SOUND_TIMEOUT: 5000,
+	BLOCK_SOUND_TIMEOUT: 2000,
```

Comment:
I think it is a mix of both. It was mostly to prevent overlapping, but also came in handy to not have too many sounds played. During the coworking I also favored staying at 5s

Reply:
2500??? :)